### PR TITLE
docs: add RFC(Introduction)

### PR DESCRIPTION
**Pull request by hi-rustin**
part of https://github.com/tispace-dev/tispace/issues/62

[rendered](https://github.com/hi-rustin/tispace/blob/rustin-patch-rfc/docs/src/intro.md)

### CODE REVIEW

File: docs/src/intro.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Motivation
+
+At PingCAP, our developers use a lot of virtual machines for our development and testing work. However, these virtual machines have poor performance and they are heavily overdraft. In addition, at PingCAP we do not have a good management platform to distribute and manage these VMs for developers. Developers manage and request these machine resources by asking each other. 
+
+To optimize these inefficient processes, we decided to develop a management platform to efficiently manage and allocate machine resources.
+
+## Detailed design
+
+The whole system uses [K8s] as the infrastructure, and deploys [ceph] clusters in [K8s] for disk high availability, and replaces the [K8s] container layer with [kata] to provide good virtualization capability.
```
Comment (gozssky) [inline docs/src/intro.md:23]: `replaces` may be `choose`. K8s supports a wide variety of container runtimes. cri-o with kata is one of them.

File: docs/src/intro.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### K8s
+
+We have deployed [K8s] clusters on bare machine and will rely on [K8s] to provide the scheduling capabilities for the entire system. With this, users only need to provide resource requests when creating instances, and we can use [K8s] scheduling capabilities to create the corresponding Pod resources for users.
+
+### ceph
+
+[Ceph] is a distributed storage system that supports object, block, and file storage interfaces. We built a [ceph] cluster using [Rook] in the [K8s] cluster, using it as a persistent volume resource, and assigning it to each Pod through a persistent volume claim when a user requests an instance. This way, the user's data is automatically guaranteed to be highly available, and the disk resources have the ability to be scheduled.
+
+### kata
```
Comment (gozssky) [inline docs/src/intro.md:35]:
```suggestion
### Kata
```

File: docs/src/intro.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+The whole system uses [K8s] as the infrastructure, and deploys [ceph] clusters in [K8s] for disk high availability, and replaces the [K8s] container layer with [kata] to provide good virtualization capability.
+
+<!-- TODO: Add architecture diagram -->
+
+### K8s
+
+We have deployed [K8s] clusters on bare machine and will rely on [K8s] to provide the scheduling capabilities for the entire system. With this, users only need to provide resource requests when creating instances, and we can use [K8s] scheduling capabilities to create the corresponding Pod resources for users.
+
+### ceph
```
Comment (gozssky) [inline docs/src/intro.md:31]:
```suggestion
### Ceph
```

File: docs/src/intro.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## Detailed design
+
+The whole system uses [K8s] as the infrastructure, and deploys [ceph] clusters in [K8s] for disk high availability, and replaces the [K8s] container layer with [kata] to provide good virtualization capability.
+
+<!-- TODO: Add architecture diagram -->
+
+### K8s
+
+We have deployed [K8s] clusters on bare machine and will rely on [K8s] to provide the scheduling capabilities for the entire system. With this, users only need to provide resource requests when creating instances, and we can use [K8s] scheduling capabilities to create the corresponding Pod resources for users.
```
Comment (gozssky) [inline docs/src/intro.md:29]: We may need to explain how to make modifications on rootfs persistent (like "mount persistent volume as container rootfs"). But I'm not sure where to place it, K8s or Ceph?

**Review by gozssky — COMMENTED** [review]
rest LGTM

File: docs/src/intro.md (hunk 4 above)
Comment (hi-rustin) [reply docs/src/intro.md:29]: Added.

**Review by gozssky — APPROVED** [review]

**Pull request merged**